src/containers/Storage/utils/index.ts

```diff
@@ -84,12 +85,21 @@ export function getStorageGroupsInitialEntitiesCount(
     return DEFAULT_ENTITIES_COUNT;
 }
 
-export function useVDisksWithDCMargins(vDisks: PreparedVDisk[] = []) {
+function isErasureWithDifferentDC(erasure?: Erasure) {
+    return erasure === 'mirror-3-dc' || erasure === 'mirror-3of4';
```

Comment:
what does these magic constant strings mean?

comment would be nice (and may be move to actual constants)